src/frontend/Komponenter/Personoversikt/Dokumenter.tsx

```diff
@@ -19,21 +19,37 @@ import { journalstatusTilTekst } from '../../App/typer/journalforing';
 const DokumenterVisning = styled.div`
     display: flex;
     flex-direction: column;
-
     margin-bottom: 5rem;
 `;
 
 const TrHoveddokument = styled.tr`
     background-color: #f7f7f7;
 `;
 
-const LenkeVenstrePadding = styled(Lenke)`
+const HovedLenke = styled(Lenke)<{ erklikket: boolean }>`
+    color: ${(props) => props.erklikket && '#634689'};
+`;
+
+const LenkeVenstrePadding = styled(Lenke)<{ erklikket: boolean }>`
     padding-left: 2rem;
+    color: ${(props) => props.erklikket && '#634689'};
 `;
 
 const Dokumenter: React.FC<{ personopplysninger: IPersonopplysninger }> = ({
     personopplysninger,
 }) => {
+    const [klikkedeLenker, settKlikkedeLenker] = useState<Set<string>>(new Set());
+
+    const settKlikketLenke = (lenkeId: string) => {
```

Comment:
Er det meningen at dette skal funke på tvers av sesjoner? isåfall vil ikke dette løse problemet - da må vi bytte ut "Lenke" med en `<a href...`

Reply:
Det står spesifisert i Favro-kortet at de skal nullstilles ved for eksempel en refresh av siden, så da tenker jeg dette er en grei løsning i første omgang

Reply:
fixd 4b7f893